src/main/java/me/moodcat/mood/Mood.java

```diff
@@ -0,0 +1,56 @@
+package me.moodcat.mood;
+
+import lombok.Getter;
+import me.moodcat.database.embeddables.VAVector;
+
+/**
+ * A mood represents a vector in the valence-arousal plane which will be attached to song.
+ *
+ * @author JeremybellEU
+ */
+public enum Mood {
+
+    ANGRY(0.0, 0.0),
```

Comment:
Can you put some estimated values here? Given that we range in [-1, 1] and take the points from the graph in the book?